languages/csharp/config.json

```diff
@@ -24,6 +24,12 @@
         "concepts": ["booleans"],
         "prerequisites": ["basics"]
       },
+      {
+        "slug": "casting",
```

Comment:
I think this should be part of another PR?

Reply:
`casting` is in master but was not in the base of this branch.  Won't it merge as is or do I have to do  something with my branch.  Last time I did anything like that it was a disaster and the PR says This Branch has no Conflicts.

Reply:
Ah, I didn't realize it was in master. Just keep it in.